indico/modules/events/papers/blueprint.py

```diff
@@ -36,6 +36,9 @@
                  methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/papers/enable/<any(content,layout):reviewing_type>', 'switch',
                  management.RHSwitchReviewingType, methods=('PUT', 'DELETE'))
+_bp.add_url_rule('/manage/papers/assignment-list', 'assignment', management.RHPapersAssignmentList)
+_bp.add_url_rule('/manage/papers/assign/customize', 'customize_assignment_list', management.RHAssignmentListCustomize,
```

Comment:
I would use `/manage/papers/assignment-list/customize` here and in the line above `/manage/papers/assignment-list/` (slash at the end)